service/surf/apps/materials/utils.py

```diff
@@ -197,6 +209,7 @@ def add_search_query_to_elastic_index(number_of_results, query, filters):
         create_search_results_index(elastic.client)
 
     document = {
+        'timestamp': datetime.datetime.utcnow().replace(tzinfo=datetime.timezone.utc).isoformat(),
```

Comment:
Je vraagt om een utcnow en vervolgens moet je tz info vervangen met UTC? Lijkt me dubbelop, maar opzicht mag je mergen van mij.

Reply:
Volgens mij was dat nodig om de timezone in de string te krijgen. Ik zal nog een keer checken wat er gebeurt als ik het weg laat.

Reply:
@fako , verschil is:

```
met tzinfo: 2020-11-09T13:02:35.155611+00:00
zonder tzinfo: 2020-11-09T13:02:35.155662
```

Reply:
Thanks! Weet ik dat ook weer :)